src/features/exports.test.ts

```diff
@@ -133,12 +133,18 @@ describe.concurrent('generateExports', () => {
       },
       {},
     )
-    await expect(results).resolves.toMatchInlineSnapshot(`
+    expect(results).toMatchInlineSnapshot(`
       {
         "exports": {
           ".": {
-            "import": "./foo.js",
-            "require": "./foo.cjs",
+            "import": {
+              "default": "./foo.js",
+              "types": "./foo.d.ts",
```

Comment:
Still incorrect

Reply:
Since writeFile also uses JSON.stringify, key order validation is handled in the following line:
expect(JSON.stringify(results.exports, undefined, 2)).toMatchInlineSnapshot(...).